spec/document_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Kitchen::Document do
+  let(:book1) do
+    new_element(
+      <<~HTML
+        <div>
+          <div data-type="chapter" id="chapter1">
+            <span>This is a chapter</span>
+          </div>
+        </div>
+      HTML
+    )
+  end
```

Comment:
`book1` is actually a `BookDocument`, which yes is a `Document` but I think it'd be better to just instantiate a `Kitchen::Document` directly with a dummy Nokogiri document.  

Also below is saying `book1.document` everywhere -- `book1` is never used so it shouldn't be what is returned, better to have `let(:document)`